packages/builder-rsbuild/src/types.ts

```diff
@@ -47,6 +47,10 @@ export type BuilderOptions = {
    * Which environment to use from the Rsbuild config.
    */
   environment?: string
+  /**
+   * @storybook/addon-docs options
+   */
+  addonDocs?: any
```

Comment:
Can't find a way to do better than this given the typings of `@storybook/addon-docs/dist/preset`:

```ts
import { PresetProperty, Options } from 'storybook/internal/types';

declare const addons: PresetProperty<'addons'>;
declare const viteFinal: (config: any, options: Options) => Promise<any>;
declare const webpackX: any;
declare const docsX: any;
/**
 * If the user has not installed react explicitly in their project,
 * the resolvedReact preset will not be set.
 * We then set it here in addon-docs to use addon-docs's react version that always exists.
 * This is just a fallback that never overrides the existing preset,
 * but ensures that there is always a resolved react.
 */
declare const resolvedReact: (existing: any) => Promise<{
    react: any;
    reactDom: any;
    mdx: any;
}>;
declare const optimizeViteDeps: string[];

export { addons, docsX as docs, optimizeViteDeps, resolvedReact, viteFinal, webpackX as webpack };
```

No type export for the options directly, and the`webpack` export is `any`, preventing us to extract it's parameters.